dashboard/client/src/service/node.ts

```diff
@@ -1,10 +1,18 @@
-import { NodeDetailRsp, NodeListRsp } from "../type/node";
+import {
+  NodeDetailRsp,
+  NodeListRsp,
+  NodeWithWorkersListRsp,
+} from "../type/node";
 import { get } from "./requestHandlers";
 
 export const getNodeList = async () => {
   return await get<NodeListRsp>("nodes?view=summary");
 };
 
+export const getNodesWithWorkers = async () => {
+  return await get<NodeWithWorkersListRsp>("nodes?view=details");
```

Comment:
Can we think more about the large scale scenes here? If we deploy the Ray cluster in a large scale, we will fetch a big detail table by this API. So, can we make it to be interactive? When the users click a node, we fetch the details for only one node on demand.

Reply:
The main concern for doing this on demand, is that we fetch regularly to update the UI. If a user has multiple nodes expanded, then we will be making multiple fetches instead of a single one. 

That being said, when all the nodes are expanded, it can be pretty difficult to view everything at once so maybe users don't expand too many items at once.

Reply:
okay, i changed to fetch only for expanded nodes